addons/html_editor/static/src/main/media/media_dialog/file_media_dialog.js

```diff
@@ -1,61 +0,0 @@
-import { renderToElement } from "@web/core/utils/render";
```

Comment:
Stable retrocompatibility: Same. Put this file back with a comment on top saying it is not used anymore but was kept due to the stable policy.